mim/utils/utils.py

```diff
@@ -79,12 +79,12 @@ def wrapper(package):
     return wrapper
 
 
-@ensure_installation
+# @ensure_installation
```

Comment:
May delete this line rather than comment it.

Reply:
`ensure_installation` is required and I forgot to remove the `#`.